views/scoring.erb

```diff
@@ -0,0 +1,6 @@
+<!-- displays scores after word entered -->
+<% unless display.nil? %>
```

Comment:
Using a local variable is great - if someone only has the context of this view, it is not clear what `display` is actually referring to